### ISSUE TITLE
Was it a clerical error ? ScaleNorm.g init form dim ** -0.5.  I think it should be dim ** 0.5

### ISSUE THREAD

**Issue by junphine** [lucidrains/x-transformers#246]
class ScaleNorm(nn.Module):
    def __init__(self, dim, eps = 1e-5):
        super().__init__()
        self.eps = eps
        self.g = nn.Parameter(torch.ones(1) * (dim ** -0.5))

**Comment by lucidrains** [comment]
@junphine hey, thank you for catching this! indeed the sign was not correct

it should be identical to rmsnorm except it is a single learned parameter rather than the model feature dimension
